<launch>

    <rosparam command="load" file="$(find sbmpo_ros)/config/states.yaml" ns="sbmpo"/>
    <rosparam command="load" file="$(find sbmpo_ros)/config/config.yaml" ns="sbmpo"/>

    <node pkg="sbmpo_ros" type="sbmpo" name="sbmpo" output="screen">
        <param name="map_topic" value="/map"/>
        <param name="path_topic" value="/path"/>
        <param name="poses_topic" value="/poses"/>
        <param name="publish_all" value="true"/>
        <param name="publish_rate" value="1"/>
    </node>
    
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find sbmpo_ros)/config/map.rviz" />

</launch>